org.bitcoincore.bitcoin-qt.json

```diff
@@ -0,0 +1,93 @@
+{
+    "id": "org.bitcoincore.bitcoin-qt",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "18.08",
+    "command": "bitcoin-qt",
+    "finish-args": [
+	/* Wayland and Xorg support */
+	"--socket=x11",
+	"--socket=wayland",
+	/* Sound */
+	"--socket=pulseaudio",
+	/* OpenGL access. */
+	"--device=dri",
+	/* IPC access. */
+	"--share=ipc",
+	/* Network acess. */
+	"--share=network"
+    ],
+    "modules": [
+	{
+	    "name": "bitcoin-core",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"install -m 0744 -D -T bin/bitcoind     /app/bin/internal_bitcoind",
+		"install -m 0744 -D -T bin/bitcoin-qt   /app/bin/internal_bitcoin-qt",
+		"install -m 0744 -D -T bin/bitcoin-cli  /app/bin/internal_bitcoin-cli",
+		"install -m 0744 -D -t /app/bin bin/test_bitcoin"
+	    ],
+	    "post-install": [
+		"/app/bin/test_bitcoin"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "only-arches": ["x86_64"],
+		    "url": "https://bitcoincore.org/bin/bitcoin-core-0.18.1/bitcoin-0.18.1-x86_64-linux-gnu.tar.gz",
+		    "sha256": "600d1db5e751fa85903e935a01a74f5cc57e1e7473c15fd3e17ed21e202cfe5a"
+		},
+		{
+		    "type": "archive",
+		    "only-arches": ["i386"],
+		    "url": "https://bitcoincore.org/bin/bitcoin-core-0.18.1/bitcoin-0.18.1-i686-pc-linux-gnu.tar.gz",
+		    "sha256": "989e847b3e95fc9fedc0b109cae1b4fa43348f2f712e187a118461876af9bd16"
+		},
+		{
+		    "type": "archive",
+		    "only-arches": ["arm"],
+		    "url": "https://bitcoincore.org/bin/bitcoin-core-0.18.1/bitcoin-0.18.1-arm-linux-gnueabihf.tar.gz",
+		    "sha256": "cc7d483e4b20c5dabd4dcaf304965214cf4934bcc029ca99cbc9af00d3771a1f"
+		},
+		{
+		    "type": "archive",
+		    "only-arches": ["aarch64"],
+		    "url": "https://bitcoincore.org/bin/bitcoin-core-0.18.1/bitcoin-0.18.1-aarch64-linux-gnu.tar.gz",
+		    "sha256": "88f343af72803b851c7da13874cc5525026b0b55e63e1b5e1298390c4688adc6"
+		}
+
+	    ]
+	},
+	{
+	    "name": "misc",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"install -Dp -m 644 org.bitcoincore.bitcoin-qt.desktop /app/share/applications/org.bitcoincore.bitcoin-qt.desktop",
+		"install -Dp -m 644 bitcoin.svg /app/share/icons/hicolor/scalable/apps/org.bitcoincore.bitcoin-qt.svg",
+		"install -Dp -m 644 org.bitcoincore.bitcoin-qt.appdata.xml /app/share/metainfo/org.bitcoincore.bitcoin-qt.appdata.xml",
+		"install_wrappers.sh"
```

Comment:
You need to exec the file here. `/bin/sh: install_wrappers.sh: command not found`